### PR TITLE
Modifies the `import-data` management command

### PR DESCRIPTION
**Pull request by kurtw**
This makes some changes to the `import-data` command that makes it more maintainable, object-oriented, pythonic and adds the ability to import data that won't be fully fledged pages into what wagtail calls snippets. [Here's some documentation on snippets](http://docs.wagtail.io/en/v1.1/topics/snippets.html). These are things that we want to define once in the CMS but use in multiple areas e.g. Contacts. 

There's also a change to the way that we execute the command.
```
usage: manage.py import-data [-h] [--version] [-v {0,1,2,3}] [--settings SETTINGS] [--pythonpath PYTHONPATH] [--traceback] [--no-color] [--parent PARENT] [--snippet] -u
                             USERNAME -p PASSWORD [--app APP] [--overwrite]
                             data_type wagtail_type
```
**Note** that the `--parent` has become an option, and that the `--parent` or `--snippet` option must be used for the command to execute. The `--parent` option requires the name of the parent's slug whereas the `--snippet` option only needs to be present, as the `wagtail_type` parameter would be the name of the snippet.

Here's an example of importing pages:
```
./cfgov/manage.py import-data events eventpage --parent events -u username -p pass
```
And here's an example of importing snippets:
```
./cfgov/manage.py import-data contact contact --snippet -u username -p pass
```

One more **BIG** thing. It's not in this PR right now, but it will be necessary that the processor file for snippets include both a function called **convert(imported_data)** (which is also necessary for a page processor) and a function called **get_existing_snippet(imported_data)** that accepts one argument (the imported data) and will return either the snippet that already exists based the imported data or `None` if there isn't an existing one.

## Additions

- Adds new file to commands module in the core app called _helpers.py
- Adds ability to import snippets

## Changes

- Abstracts code into helper class DataImporter
- Modifies command line options to allow specifying arguments for importing pages or snippets
- Changes the way the processor module is imported so it imports it using the [app] argument
- Moves the processors module from the core.management.commands module to the v1 app

## Testing

- Run it. Unfortunately, the snippet functionality can't be tested until a snippet's processor is available (soon).

## Review

- @rosskarchner 
- @kave 
- @anselmbradford 
- @KimberlyMunoz 
- @sebworks 
- @jimmynotjim 

## Notes

- It's entirely possible for migrating snippets to require some editing after a

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -84,6 +84,8 @@ Given the `MAJOR.MINOR.PATCH` pattern, here is how we decide to increment:
 - Added template for post-preview molecule
 - Added templates and CSS for the Signup Form organism.
 - Added templates and CSS for the Content Sidebar organism.
+- Adds new file to commands module in the core app called _helpers.py
```
Comment (anselmbradford) [inline CHANGELOG.md]: Put this in ticks to avoid the markdown italicizing `_helpers.py`

**Comment by anselmbradford** [comment]
:+1: Not entirely sure what I'm looking at, but nothing jumped out at me.

**Comment by kurtw** [comment]
I wanted to tag all the devs to make sure everyone knew that this was changing.

**Comment by kurtw** [comment]
Made some updates but need still need to finalize README before merge.

**Comment by jimmynotjim** [comment]
Ditto Ans :+1:

**Comment by kave** [comment]
:+1: looks good, i'll create a processor and test the functionality

**Pull request merged**